@navikt/core/tokens/darkside/tokens/semantic-roles.ts

```diff
@@ -155,8 +122,23 @@ const configForRole = (role: GlobalColorRoles, theme: ColorTheme) => {
 /**
  * We need to deep merge the token config for each role to get the complete token config for all roles.
  */
-export const semanticTokensForAllRolesConfig = (theme: ColorTheme) =>
-  globalColorRoles.reduce(
+export const semanticTokensForAllRolesConfig = (theme: ColorTheme) => {
+  const roles: ColorRoles[] = [
+    "neutral",
+    "accent",
+    "success",
+    "warning",
+    "danger",
+    "info",
+    "brand-magenta",
+    "brand-beige",
+    "brand-blue",
+    "meta-purple",
+    "meta-lime",
+  ] as const;
```

Comment:
Generally I don't like the fact that you can add items to the type without getting an error if you forget to add it here, but I guess the chance of that happening is low.